deploy/charts/cert-manager/values/go.mod

```diff
@@ -0,0 +1,7 @@
+module main
+
+go 1.16
+
+replace github.com/alecthomas/jsonschema => github.com/amurant/jsonschema v0.0.0-20210729111349-9c856f912aa1
```

Comment:
Maybe add a comment explaining the changes in your fork and linking to a PR where you attempt to get these accepted upstream.

Reply:
upstreamed already